backend/packages/db/arangodb.config.js

```diff
@@ -0,0 +1,90 @@
+// WARNING - changing this file will mutate the structure of the database.
```

Comment:
This config allows us to declaratively define the database topology. Be careful when editing it.